it/test/controllers/agent/manageBusinesses/add/AddBusinessTradeControllerISpec.scala

```diff
@@ -85,7 +85,7 @@ class AddBusinessTradeControllerISpec extends ComponentSpecBase {
         IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, noPropertyOrBusinessResponse)
 
         When(s"I call GET ${addBusinessTradeControllerShowUrl}")
-        val result = IncomeTaxViewChangeFrontend.getAddBusinessTrade(clientDetailsWithConfirmation)
+        val result = IncomeTaxViewChangeFrontend.getAddBusinessTradeNewJourney(clientDetailsWithConfirmation)
```

Comment:
should fix : can we not use `new` in names?. It will make no sense when old ones are removed.